script/cibuild

```diff
@@ -8,54 +8,42 @@ fi
 
 failed=0
 
-if [ x"$SPEC" = "xtrue" ]; then
```

Comment:
Nice cleanup, removing the `$SPEC`

Reply:
Ta! It was always ugly; really old, too.